src/core/Akka.Remote/Configuration/Remote.conf

```diff
@@ -12,7 +12,7 @@ akka {
   actor {
 	serialization-settings {
       primitive {
-        use-neutral-primitives = off
+        use-legacy-behavior = on
```

Comment:
Yep, good idea - need to have this set to `on` by default so users don't hit the issue while upgrading.